packages/core/src/attestation/Attestation.ts

```diff
@@ -60,10 +60,18 @@ export default class Attestation implements IAttestation {
    * @param maxDepth - The number of levels to walk up the delegation hierarchy until the delegation node is found.
    * @returns A promise containing the unsigned SubmittableExtrinsic (submittable transaction).
    * @example ```javascript
-   * Attestation.revoke('0xd8024cdc147c4fa9221cd177', 3).then(() => {
-   *   // the attestation revocation tx was created, sign and send it!
-   *   BlockchainUtils.signAndSendTx(tx, identity);
-   * });
+   * Attestation.revoke('0xd8024cdc147c4fa9221cd177', 3).then(
+   *   (revocationExtrinsic) => {
+   *     // The attestation revocation tx was created, and it can now be signed by the attestation owner.
+   *     const attestationOwnerDid = did;
```

Comment:
C: This does not seem to be needed

Reply:
https://github.com/KILTprotocol/sdk-js/pull/434/commits/974293be648c3fafc24709faf09638530614cb5c